media/docs/cpp/build/building_with_sycl_support.md

```diff
@@ -49,7 +30,18 @@ Building the tests and the examples requires oneMKL for random number generation
 
 The following instructions show how to use the nightly build to build the cutlass examples
 
+#### Building with oneAPI Basekit
+Download [oneAPI basekit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html)
+```
+$ . /opt/intel/oneapi/setvars.sh
+$ CC=icx CXX=icpx cmake .. -G Ninja \
+  -DCUTLASS_ENABLE_SYCL=ON \
+  -DDPCPP_SYCL_TARGET=intel_gpu_pvc \
+  -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+  -DCMAKE_CXX_FLAGS="-ftemplate-backtrace-limit=0 -fdiagnostics-color=always"
+```
 
+#### Building with release DPCPP compiler
```

Comment:
don't think we should call it "release DPC++". I suggest something like "open source version of DPC++" or "open source nightly release of DPC++" . The key difference is that the oneAPI compiler is the proprietary compiler product and this is the open-source foundation of the product.